core/src/main/java/com/graphhopper/routing/util/EncodingManager.java

```diff
@@ -510,6 +511,21 @@ public boolean canSkip() {
         }
     }
 
+    /**
+     * Determine whether a relation is routable for one of the added encoders.
+     *
+     * @return if at least one encoder consumes the specified relation. Additionally the specified acceptRelation is changed
```

Comment:
This javadoc is outdated I think ? Something like

```java
/**
 * @return true if any of the encoders accepts the given turn relation
 * @see FlagEncoder#acceptRelation(OSMTurnRelation)
 */
```

would be enough. Or just remove the entire method in case its not needed.

Reply:
I remove it too.